### DISCUSSION TITLE
RuntimeWarning

### DISCUSSION THREAD

**Discussion by Wuwei-Mo** [pycalphad/pycalphad#489] [Q&A] [answered]
Hi,

I often do thermodynamic calculations using Pycalphad with different TDB files, but I also often see a warning. So I'm asking if this warning causes inaccurate results when it occurs, or should I just ignore it.
For example：

dbf = Database(r'TDB/Al-Cu-Mg-Si.TDB')
phases = dbf.phases.keys()
elements_pycalphad = ['AL','CU','MG','SI','VA']
Temp = 500
eq = equilibrium(dbf, elements_pycalphad, phases,  
                 {v.X('CU'):0.1, v.X('MG'):0.1, v.X('SI'):0.1, 
                  v.T: Temp, v.P:101325}, output='HM')

![1](https://github.com/pycalphad/pycalphad/assets/129748194/5517ba7a-5dde-4a49-820b-7172bf3a2c37)

**Comment by richardotis** [comment, marked as answer]
I thought we had merged a fix for this some time ago, but apparently not. This warning can be safely ignored. It has to do with a division by zero for a term that is already zero, so it doesn't affect the result in the sum.

**Reply by richardotis** [reply]
I see, we had talked about fixing it in https://github.com/pycalphad/pycalphad/issues/449 but haven't gotten to it yet.

**Reply by Wuwei-Mo** [reply]
Ok, thanks
